src/uct/ib/base/ib_iface.c

```diff
@@ -634,20 +604,25 @@ struct ibv_cq *uct_ib_create_cq(struct ibv_context *context, int cqe,
 #else
     cq = ibv_create_cq(context, cqe, NULL, channel, comp_vector);
 #endif
-    return cq;
+    if (!cq) {
+        ucs_error("ibv_create_cq(cqe=%d) failed: %m", cqe);
+        return UCS_ERR_IO_ERROR;
+    }
+
+    *cq_p = cq;
+    return UCS_OK;
 }
 
-static ucs_status_t uct_ib_iface_create_cq(uct_ib_iface_t *iface, int cq_length,
-                                           size_t *inl, int preferred_cpu,
-                                           int flags, struct ibv_cq **cq_p)
+static ucs_status_t __uct_ib_iface_create_cq(uct_ib_iface_t *iface, int cq_length,
```

Comment:
minor: I'd remove `__` from the name and change existing `uct_ib_iface_create_cq` to something like `uct_ib_verbs_iface_create_cq`.